lib/dotcom/system_status.ex

```diff
@@ -0,0 +1,32 @@
+defmodule Dotcom.SystemStatus do
+  @moduledoc """
+  Parent module for the system status feature
+  """
+
+  alias Dotcom.SystemStatus
+
+  @doc """
+  Returns a list of alerts that satisfy the following criteria:
+  - They are for one of the subway or trolley lines (including Mattapan), and
+  - They are either currently active, or will be later today
```

Comment:
We could make this more straightforward by saying that the alert is active today... if it is currently active, it will necessarily be active today.

Reply:
I wrote it the way I did because "active today" sort of implies that it would also include alerts that were active earlier today, but have been closed out. I guess I could write it as
```
- They are active today, and not in the past
```

Or maybe
```
- They will be active at some point later today (possibly including now)
```